x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -78,7 +78,7 @@ export function DetailView({ errorGroup, urlParams, location }: Props) {
   }
 
   const tabs = getTabs(error);
-  const currentTab = getCurrentTab(tabs, urlParams.detailTab);
+  const currentTab = getCurrentTab(tabs, urlParams.detailTab) as any;
```

Comment:
Can we avoid `any`? `getCurrentTab` should return a type `ErrorTab`. Does it help adding that returntype to `getCurrentTab`?

Reply:
Solved I think